NoSoft-SA/nspack
# New PDT app to clone cartons

Summary of this change:
App: autofill_pallet_cartons
    Scan pallet or carton
    System should prompt the user whether X cartons should be created (cpp minus pallet.carton_qty)
    If yes, call existing autofill service, log a different status AUTO_FILLED_<X>_CARTONS_BY_PDT

### Added
- . PDT App to autofill pallet cartons

How to test this code via the UI:
RMD | Production | Palletizing | Autofill Pallet Cartons

 * Scan pallet or carton
 * Click Autofill
 * Prompt the user whether X cartons should be created (cpp minus pallet.carton_qty)
 *  If yes, call existing autofill service, log a different status AUTO_FILLED_<X>_CARTONS_BY_PDT
